app/javascript/dashboard/components/widgets/WootWriter/ReplyBottomPanel.vue

```diff
@@ -116,29 +116,27 @@ export default {
   setup() {
     const { setSignatureFlagForInbox, fetchSignatureFlagFromUISettings } =
       useUISettings();
+
     const uploadRef = ref(null);
     // TODO: This is really hacky, we need to replace the file picker component with
     // a custom one, where the logic and the component markup is isolated.
     // Once we have the custom component, we can remove the hacky logic below.
-    const uploadTriggerButton = computed(() => {
-      if (uploadRef.value) {
-        return uploadRef.value.$children[1].$el;
-      }
-
-      return null;
-    });
+    const uploadRefElem = computed(() => uploadRef.value?.$el);
```

Comment:
Why do we need this separate ref?

Reply:
@scmmishra If we pass the `uploadRef.value` directly, then the event will not add.
Here is how it looks when we pass `uploadRef.value` directly.
![image](https://github.com/user-attachments/assets/bc39e1f5-7128-4a0f-a2df-feafeb130438)